wordpress.org/public_html/wp-content/plugins/wporg-gp-custom-errors/wporg-gp-custom-errors.php

```diff
@@ -15,6 +15,20 @@ class WPorg_GP_Custom_Translation_Errors {
 	 */
 	const WORPRESS_CORE_PROJECT_ID = 1;
 
+	/**
+	 * The parent WordPress core project ID.
+	 *
+	 * @var int
+	 */
+	const CONTEXT_ENUM_CHECK_LIST = [
+		'text direction' => [ 'ltr', 'rtl' ],
+		'Open Sans font: on or off' => [ 'on', 'off' ],
+		'Open Sans font: add new subset (greek, cyrillic, vietnamese)' => [ 'no-subset', 'greek', 'cyrillic', 'vietnamese' ],
+		'Word count type. Do not translate!' => [ 'characters_excluding_spaces', 'characters_including_spaces', 'words' ],
+		'decline months names: on or off' => [ 'on', 'off' ],
+		'Comment number declension: on or off' => [ 'on', 'off' ],
+	];
```

Comment:
These in general follow a specific format, I wonder if we could add something more generic.

 - `Word: ... (option1, option2)` = must be option1 or option2
 - `Word: option1 or option2` = Must be option1 or option2
 - `Do not translate` in the comment, and neither of the above = `Triple check please`?

Wouldn't cover 100% of cases, but would help with such strings in the future.